docs/source/man.rst

```diff
@@ -559,6 +559,16 @@ The ``qpc report merge`` command combines results from two or more scan job iden
 
   A Path to a directory with JSON details report files that will be merged. Mutually exclusive with the ``--job-ids`` and the ``--report-ids`` option.
 
+Report Merge Status
+~~~~~~~~~~~~~~~~~~~
+The ``qpc report job`` command can be used to check the status of a large merge of JSON details report files. The ``qpc report merge --json-directory`` can be used to generate a job id.
```

Comment:
Change the last sentence to:
A large merge is created with the ``qpc report merge --json-directory=path_to_directory_of_json_files`` command. This command will return the merge job id, which can be used to access the status of the job.